test/algorithms/simulation/sidb/quicksim.cpp

```diff
@@ -74,4 +74,103 @@ TEMPLATE_TEST_CASE(
             CHECK(!it.charge_exists(sidb_charge_state::POSITIVE));
         }
     }
+
+    SECTION("zero threads")
+    {
+        TestType lyt{{20, 10}};
+
+        lyt.assign_cell_type({1, 3, 0}, TestType::cell_type::NORMAL);
+        lyt.assign_cell_type({3, 3, 0}, TestType::cell_type::NORMAL);
+        lyt.assign_cell_type({4, 3, 0}, TestType::cell_type::NORMAL);
+
+        lyt.assign_cell_type({6, 3, 0}, TestType::cell_type::NORMAL);
+        lyt.assign_cell_type({7, 3, 0}, TestType::cell_type::NORMAL);
+
+        lyt.assign_cell_type({6, 10, 0}, TestType::cell_type::NORMAL);
+        lyt.assign_cell_type({7, 10, 0}, TestType::cell_type::NORMAL);
+
+        quicksim_stats<TestType>         quicksimstats{};
+        const sidb_simulation_parameters params{2, -0.30};
+        const struct quicksim_params     quicksim_params
```

Comment:
Remove `struct` keyword. You probably don't want to define a new struct here ;) Same for the other sections

Reply:
Thank you! Weird, because before it was required when all was in one section.